components/eamxx/src/dynamics/homme/eamxx_homme_process_interface.hpp

```diff
@@ -7,6 +7,7 @@
 #include "ekat/ekat_parameter_list.hpp"
 #include "ekat/ekat_pack.hpp"
 #include "ekat/ekat_workspace.hpp"
+#include "physics/p3/p3_functions.hpp"
```

Comment:
We probably should avoid including unrelated headers.

Reply:
thanks, that seems to be a leftover from before